agent/registration/csr.go

```diff
@@ -34,10 +35,11 @@ const (
 	ByohCSRCNFormat   = "byoh:host:%s"
 	ByohCSRNameFormat = "byoh-csr-%s"
 	TmpPrivateKey     = "byoh-client.key.tmp"
+	DefaultConfigPath = ".byoh/config"
```

Comment:
can't we define the default as `~/.byoh/config`?

Reply:
That's what was originally there, but we need to use `os.UserHomeDir()` to fetch home dir. This is now being done in `GetBYOHConfigPath()` function.